src/main/java/com/synopsys/integration/alert/component/settings/descriptor/SettingsUIConfig.java

```diff
@@ -95,6 +100,9 @@
     private static final String SETTINGS_SAML_METADATA_URL_DESCRIPTION = "The Metadata URL from the external Identity Provider.";
     private static final String SETTINGS_SAML_ENTITY_ID_DESCRIPTION = "The Entity ID of the Service Provider. EX: This should be the Audience defined in Okta.";
     private static final String SETTINGS_SAML_ENTITY_BASE_URL_DESCRIPTION = "This should be the URL of the Alert system.";
+    private static final String SETTINGS_SAML_METADATA_FILE_DESCRIPTION = "The file to upload to the server containing the Metadata from the external Identity Provider.";
+
+    private static final String BUTTON_LABEL_SAML_METADATA_FILE = "Upload Metadata File";
```

Comment:
Since this seems to be the button text, should we simplify this to simply upload? The label should do the job of describing